### ISSUE TITLE
'brew install osquery' is still featured on osquery downloads page

### ISSUE THREAD

**Issue by peterlewis** [osquery/osquery-site#172]
Howdy!

I gather from reading other issues/PRs that you no longer provide/maintain a Homebrew formula for osquery v4.x.

I've observed that at https://osquery.io/downloads/official/4.3.0 it still mentions:

```
We publish a stable osquery into OS X homebrew every other week. Installing using brew has several advantages: most current stable release (with additional performance guarantees). You should update brew often.

$ brew update
$ brew install osquery
$ /usr/local/bin/osqueryi
```

Please can I make a recommendation that this is either removed or replaced it with a reference to the Homebrew Cask that is [now available](https://github.com/Homebrew/homebrew-cask/pull/84440)?

The latter may be less-required/suitable, as all it does is install the .pkg, but still, thought I'd mention it as an option! :)

Thanks!

Peter

**Comment by theopolis** [comment]
Thanks for following up here!

I think we still want to include guidance for those using brew. It would be awesome if you could make the change to the cask-install instructions. (If you have time.)

**Comment by peterlewis** [comment]
Happy to contribute, however I'm unsure of what the most appropriate text would be, to replace the following:

```
We publish a stable osquery into OS X homebrew every other week. Installing using brew has several advantages: most current stable release (with additional performance guarantees). You should update brew often.
```

You could simply say `osquery is available for installation from the Homebrew Cask repository` but it is a wee bit short! :)

However, in terms of the actual installation steps, though, I believe it should simply be a case of replacing:

```
"brew update",
"brew install osquery",
"/usr/local/bin/osqueryi"
```

With:

```
"brew update",
"brew cask install osquery"
```

Also, just to mention, in terms of maintaining the cask, obviously I'm happy to (and intend to!) submit a PR to update the osquery cask when I spot a new release, but it might be worth the osquery development team adding a PR to the homebrew-cask repo, in to the workflow, as new releases become available? :)

Sorry the above is attached to the issue rather than taking the form of a PR, just wanted to get your thoughts!

Thanks,

Peter

**Comment by theopolis** [comment]
> You could simply say `osquery can be installed from the Homebrew Cask repository` but it is a wee bit short! :)

I love this, direct and accurate!

> However, in terms of the actual installation steps, though, I believe it should simply be a case of replacing ...

Ship it!

> Also, just to mention, in terms of maintaining the cask, obviously I'm happy to (and intend to!) submit a PR to update the osquery cask when I spot a new release, but it might be worth the osquery development team adding a PR to the homebrew-cask repo, in to the workflow, as new releases become available? :)

Can do!

**Comment by directionless** [comment]
I don't know that anyone is working on making a cask recipe. Some background at https://github.com/Homebrew/homebrew-core/pull/54905

**Comment by peterlewis** [comment]
> I don't know that anyone is working on making a cask recipe. Some background at [Homebrew/homebrew-core#54905](https://github.com/Homebrew/homebrew-core/pull/54905)

There exists, now [a cask](https://github.com/Homebrew/homebrew-cask/pull/84440) that can be used to install osquery via Homebrew, once more! :)

**Comment by directionless** [comment]
This was fixed by #173
